components/PowerInfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import Image from 'next/image';
+
+type PowerInfo = {
+  src: string;
+  applicance: string;
+  kWh: string;
+};
+
+const PowerInfoCard: FC<PowerInfo> = ({ src, applicance, kWh }) => {
+  return (
+    <div className="w-full">
+      <div className="bg-ocf-gray-1000 pt-3 pb-8 rounded-lg flex justify-center align-middle">
```

Comment:
the spacing on this element is a bit closer than on the figma—we could probably remedy this by adding a bit of `gap`
<img width="311" alt="Screen Shot 2023-03-10 at 10 28 34 PM" src="https://user-images.githubusercontent.com/62641231/224464851-6e5ec069-4fe0-4a3c-a6b5-de95cbac0322.png">

```suggestion
      <div className="bg-ocf-gray-1000 pt-3 pb-8 rounded-lg flex justify-center align-middle gap-4">
```